src/mixxx.cpp

```diff
@@ -1502,14 +1501,19 @@ void MixxxMainWindow::rebootMixxxView() {
         int newX = initPosition.x() + (initSize.width() - m_pWidgetParent->width()) / 2;
         int newY = initPosition.y() + (initSize.height() - m_pWidgetParent->height()) / 2;
 
-        const QScreen* primaryScreen = getPrimaryScreen();
-        if (primaryScreen) {
-            newX = std::max(0, std::min(newX, primaryScreen->geometry().width() - m_pWidgetParent->width()));
-            newY = std::max(0, std::min(newY, primaryScreen->geometry().height() - m_pWidgetParent->height()));
-            move(newX,newY);
-        } else {
+        const QScreen* const pScreen = mixxx::widgethelper::getScreen(*this);
+        VERIFY_OR_DEBUG_ASSERT(pScreen) {
             qWarning() << "Unable to move window inside screen borders.";
         }
+        else {
+            const auto screenBorderWidth =
+                    pScreen->geometry().width() - m_pWidgetParent->width();
+            const auto screenBorderHeight =
+                    pScreen->geometry().height() - m_pWidgetParent->height();
```

Comment:
For Shade that is 358 and 268
For LateNight it is 86 and 100
It does not change when the I resize the Mixxx window. 

So it looks like it is no Border it is the gap between the screen border and the minimum Skin size. 

Using this value for moving the window is wrong, because we need to consider the actual size and not the minimum. 

Looking at this PR only, giving the value a misleading name is a regression. 

So we can here do one of the following to make progress: 
* Rename the variable to remainingHighFromMinimumSkinSize. 
* move back to the anonymous solution 
* Fix the code.